examples/make_error.py

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
This is nice to demonstrate, but we should remember to remove this example file before merging